publication-commons/src/test/java/no/unit/nva/publication/service/impl/ResourceServiceTest.java

```diff
@@ -1736,6 +1740,153 @@ void shouldLogExceptionWhenFailingMigratingFiles() throws BadRequestException {
                      () -> resourceService.refreshResources(List.of(Resource.fromPublication(persistedPublication))));
     }
 
+    @Deprecated
+    @Test
+    void shouldCreateFileEventWithFilePublishingDateWhenMigratingOpenFileWithPublishedDate()
+        throws BadRequestException, JsonProcessingException {
+        var json = """
```

Comment:
Using json as it is impossible to construct "published" file without publishedDate, and it is probably good thing :)